src/xcode/ENA/ENA/Source/Scenes/ErrorReporting/MainErrorReportViewControllers/BottomErrorReportView.swift

```diff
@@ -0,0 +1,217 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+import OpenCombine
+
+class BottomErrorReportView: UIView {
```

Comment:
I wonder what this has in common with the problem of the keys? This is about a bug in the coordinator, but not responsible for the scrolling problem?
That seems to be a different bug.

Reply:
> Seems for me a really breaking change. Perhaps it would be better (if not too late) to split up the refactoring from ViewController to View in a separate PR and afterwards the usage to fix the bug in another PR.
> So we eventually get a hole new bugs only for the refactoring.

Basically all I did is Change a child view controller to a view. I think the changes are rather minimal.

Reply:
I just changes this class from a view controller to a view. Nothing else